src/main/java/frc/robot/RobotContainer.java

```diff
@@ -86,9 +90,16 @@ public RobotContainer() {
   }
 
   private void configureBindings() {
-    shooter.setDefaultCommand(new InstantCommand(() -> shooter.disable(), shooter));
+    // shooter.setDefaultCommand(new InstantCommand(() -> shooter.disable(), shooter));
 
     controller.rightTrigger().whileTrue(new ShooterEnable(shooter));
+
+    controller
+        .a()
+        .whileTrue(
+            Commands.startEnd(
+                () -> shooter.runVelocity(shooterSpeedInput.get()), shooter::disable, shooter));
+
```

Comment:
Is this for testing the PID?  We should move this to "rightTrigger" eventually.

Reply:
Yes, for testing.  I want the shooter to be able to be run in open loop and closed loop by pressing different buttons.  Feel free to pick a different button than `a` if it interferes with other commands that need to be assigned to it.

Reply:
Makes sense. Fine as is. Tnx.